pkg/controller/crdmirroring/utils.go

```diff
@@ -0,0 +1,24 @@
+// Copyright 2021 Antrea Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package crdmirroring
+
+import (
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/runtime"
+)
+
+func deepCopy(obj metav1.Object) metav1.Object {
```

Comment:
Modified as you suggested.

The existence of `crdmirroring_handlers.go` is for historical reason. I wrote handler functions for every CRD and it was so long, so I put handler functions to a single file. Later I changed the code and it is much shorter now, but I didn't realize that the functions could be merged into `crdmirroring_controller.go`.  I think that merging `crdmirroring_handlers.go` into `crdmirroring_controller.go` is better.

As for `utils.go`,  putting `func deepCopy(obj metav1.Object) metav1.Object` to a single file is indeed not necessary. The reason I did so is that I think it may looks more neat.  As `crdmirroring_controller.go` is  very long,  it is okay to put them together.